src/components/app/routes/loaders/rootLoader.js

```diff
@@ -264,10 +278,21 @@ const fetchEnterpriseCuration = async (enterpriseUUID, options = {}) => {
     ...options,
   });
   const url = `${getConfig().ENTERPRISE_CATALOG_API_BASE_URL}/api/v1/enterprise-curations/?${queryParams.toString()}`;
-  const response = await getAuthenticatedHttpClient().get(url);
-  const data = camelCaseObject(response.data);
-  // Return first result, given that there should only be one result, if any.
-  return data.results[0];
+
+  try {
+    const response = await getAuthenticatedHttpClient().get(url);
+    const data = camelCaseObject(response.data);
+    // Return first result, given that there should only be one result, if any.
+    if (data.results.length > 0) {
+      return data.results[0];
+    }
+    return {};
+  } catch (e) {
+    if (e.response?.status === 404) {
+      return {};
```

Comment:
nit: imo, slight preference to `return null` in this case as consumers of these data could think the presence of this empty object means the existence of an "enterprise curation" (would require a check on one its properties).